spec/predicates/human-review-vcs.md

```diff
@@ -0,0 +1,122 @@
+
+## Use Cases
+
+Changes to source code are reviewed by other members of the development team.
+The review may be performed using systems like Gerrit or via reviewing
+mechanisms built into version control forges such as GitHub and GitLab.
+
+### Threshold of Reviews on Pull Requests
+
+GitHub allows repository owners to create protection rules that require a
+threshold of approvals before a pull request can be merged. This predicate can
+be used to express meeting such a policy. In this case, GitHub would issue a
+signed attestation identifying all the reviewers for a pull request. The
+subject of the attestation is the underlying commit at the tip of the feature
+branch. This commit, therefore, is one of the parents of the merge commit if it
+is created. If an option that requires rebasing (squash and merge / rebase and
+merge) is selected, the newly created tip commit is used as the attestation
+subject.
+
+## Model
+
+Typically, software development processes incorporate some type of code review.
+Other members of the development team are expected to study the changes
+proposed and approve them, confirming they meet various expectations pertaining
+to code quality, functionality, and so on. Therefore, the general expectation
+is that this attestation is generated as a confirmation, an approval of some
+specific change, and is performed by developers in the supply chain who have
+code review responsibilities.
+
+As this attestation is generated by a version control service or code review
+service, the signatures are from the service rather than the specific users
+issuing the review.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [
+        {
+            "name": "<REPO>",
+            "digest": { "gitCommit": "<HASH>" },
+            "uri": "<LINK TO REPO>"
+        },
+    ],
+    "predicateType": "http://in-toto.io/attestation/human-review/vcs/v0.1",
+    "predicate": {
+        "reviewers": ["<ResourceDescriptor>", ...],
+        "reviewLink": "<LINK TO REVIEW>",
+        "timestamp": "<TIMESTAMP>",
+        "annotations": {...}
+    }
+}
+```
+
+### Parsing Rules
+
+This predicate follows the
+[in-toto Attestation Framework's parsing rules](../v1/README.md#parsing-rules).
+
+### Fields
+
+`reviewers` _array of ResourceDescriptor_, _required_
+
+Indicates uniquely the identity of one or more reviewers. MUST NOT be used over
+the attestation's signature to validate the source of the attestation. This
+field enables scenarios where the attestation is issued and signed by the VCS
+review system. Each entry must include immutable information that is always
+mapped to the reviewer. The digest may be skipped in the ResourceDescriptor,
+using the name and URI fields to record the username and immutable ID.
+
+`reviewLink` _URI_, _optional_
+
+Contains a link to the full review. Useful to point to information that cannot
+be captured in the attestation.
+
+`timestamp` _Timestamp_, _optional_
```

Comment:
As above can we give this a more descriptive name?

Also note that with multiple reviewers we might want to clarify which review this refers to.  First? Last?